stories/Avatar.stories.tsx

```diff
@@ -6,6 +6,19 @@ const img1 = require('./static/avatar.png');
 const img2 = require('./static/avatar1.png');
 const img3 = require('./static/avatar2.png');
 
+const Template = (args) => {
+  return <Avatar {...args}>
+    <img src={img1} alt="avatar" />
+  </Avatar>;
+};
+
+export const Default = Template.bind({});
```

Comment:
Default is used only for Docs right? 
Just wondering if we could hide it as it's quite misleading when it appears in the list of stories in the left nav...

Reply:
Yes. Maybe we could update all of them without the Default